plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1078,4 +1078,232 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/apps/files/api/v[0-9]/transferown
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
 
+#
+# [ Nextcloud Mail ]
+#
+
+# Some email addresses end in .com or .inc
+SecRule REQUEST_FILENAME "@rx /apps/mail/api/autoconfig/ispdb/[^/]+(?:com|inc)$" \
```

Comment:
I would add also a dot at the end of regexp:
`/apps/mail/api/autoconfig/ispdb/[^/]+\.(?:com|inc)$`

Reply:
Thanks, I missed that